### ISSUE TITLE
Adding template set fails with 1.20 onwards

### ISSUE THREAD

**Issue by scshitole** [F5Networks/f5-appsvcs-templates#110]
### Environment
 * FAST RPM Version: 1.20 and 1.21
 * BIG-IP Version: 16.0.1.1

### Summary
When I add tried adding a template set to the F5 application Service template it fails giving an error as 
![image](https://user-images.githubusercontent.com/13858248/199377097-5b7dd28f-9e6d-4ab5-96ab-c6a896378ca7.png)
With 1.17, 1.18, and 1.19 there are no issues
### Steps To Reproduce
[AppRequestRouting.zip](https://github.com/F5Networks/f5-appsvcs-templates/files/9915221/AppRequestRouting.zip)

List the steps to reproduce the behavior:
Load rpm 3.36 and 1.20 and try to add  the template attached

### Expected Behavior
It should not [fail](url)
### Actual Behavior
A clear and concise description of what actually happens.
Please include any applicable error output.

**Comment by shyawnkarim** [comment]
I was able to deploy a custom template set today with v1.21.0, however, I was also able to duplicate your error.

When I use the incorrect byte size for my zip file I get the same error. Can you double check that you have the correct file size specified when uploading?

**Comment by scshitole** [comment]
@shyawnkarim how to use the correct file size? I just use Mac OS to compress the file, the same file works for 1.17 to 1.19.

**Comment by shyawnkarim** [comment]
The [Hello World Tutorial on this page](https://clouddocs.f5.com/products/extensions/f5-appsvcs-templates/latest/userguide/template-authoring.html) gives step by step instructions on how to create a custom template, zip it, send to BIG-IP, and install it. Are these the steps that you followed?

**Comment by scshitole** [comment]
@shyawnkarim I followed the steps, however the file generated is more than 1MB. I tried uploading through the GUI, it gives me same error
```
fast packageTemplateSet /Users/shitole/test/bd_epic/AppRequestRouting
Template set "AppRequestRouting" packaged as ./AppRequestRouting.zip
SEA-ML-00060393:AppRequestRouting shitole$ ls
AppRequestRouting.json	AppRequestRouting.yaml	arr.yaml
AppRequestRouting.mst	AppRequestRouting.zip
SEA-ML-00060393:AppRequestRouting shitole$ 
SEA-ML-00060393:AppRequestRouting shitole$ ls -lrt
total 96
-rw-r--r--  1 shitole  1437522721   4408 Nov  8 11:04 AppRequestRouting.json
-rw-r--r--  1 shitole  1437522721   6538 Nov  8 11:04 AppRequestRouting.yaml
-rw-r--r--  1 shitole  1437522721   3970 Nov  8 11:04 arr.yaml
-rw-r--r--  1 shitole  1437522721   6538 Nov  8 11:08 AppRequestRouting.mst
-rw-rw-rw-  1 shitole  1437522721  17771 Nov  8 11:10 AppRequestRouting.zip
SEA-ML-00060393:AppRequestRouting shitole$ 

```

**Comment by shyawnkarim** [comment]
When installing zips greater than 1MB via API, the upload needs to be broken up into chunks with multiple posts. [Instructions on how to do this can be found here.](https://support.f5.com/csp/article/K41763344#upload_large)

**Comment by scshitole** [comment]
I am not using API, I am directly using BIG-IP GUI to import the template set

**Comment by shyawnkarim** [comment]
Gotcha. There is a known GUI bug that is affecting 1.19.0 and 1.20.0 when uploading templates.  The bug has already been fixed and will be released with version 1.22.0.

**Comment by shyawnkarim** [comment]
Closing.

This issue was resolved with [Release 1.22.0](https://github.com/F5Networks/f5-appsvcs-templates/releases/tag/v1.22.0).
